tests/ctst/features/azureArchive.feature

```diff
@@ -128,3 +128,28 @@ Feature: Azure Archive
         |               Versioned |           2 |        100 |           2 |
         |               Suspended |           2 |        100 |           2 |
 
+    @2.7.0
+    @PreMerge
+    @Flaky
+    @AzureArchive
+    Scenario Outline: Failed restore objects from tar must be retried and restored
+    Given a "<versioningConfiguration>" bucket
+    And a transition workflow to "e2e-azure-archive" location
+    And <objectCount> objects "obj" of size <objectSize> bytes
+    Then object "obj-1" should be "transitioning" and have the storage class "e2e-azure-archive"
+    And object "obj-2" should be "transitioning" and have the storage class "e2e-azure-archive"
+    And manifest containing object "obj-1" should "contain" object "obj-2"
+    When i restore object "obj-1" for <restoreDays> days
+    Then blob for object "obj-1" must be rehydrated "without" notifying the queue
```

Comment:
> blob for object "obj-1" must be rehydrated "without" notifying the queue

This does not really describe the scenario.
"without notifying" is actually not an observable outcome of this scenario, but more a detail of the implementation of the "failure".

Since the scenario is "faile restore must be retried", it should look something the following, where the scenario describes the _intent_ and code deals with the gory details of "how do we actually do/simulate this":

```
When i restore object "obj-1" for <restoreDays> days
    And object "obj-1" fails to rehydrate
    Then object "obj-1" should be "transitioning" and have the storage class "e2e-azure-archive"
    And an entry should be added to dead-letter-queue

When i run sorbetctl to retry failed restore for "e2e-azure-archive" location
    Then object "obj-1" should be "restored" and have the storage class "e2e-azure-archive"
    And object "obj-1" should expire in <restoreDays> days
    And object "obj-1" should have the same data
```

Reply:
>     And object "obj-1" fails to rehydrate
I am not sure about this step, as this will just do nothing at all..

>     And an entry should be added to dead-letter-queue
This step seems a bit to "technical" to me, I think in such scenarios we don't want to know how it works internally, just the fact that it fails and it can be retried... no ?


I'm thinking of a way to make in clearer